### PR TITLE
Fix font breaking issues

### PR DESCRIPTION
**Pull request by HakkyuKim**
~~I'm still looking into other options, but this is the only working solution I have.~~ With this change, the engine must be built without the `--enable-fontconfig` option.

With fontconfig, Rendering SamsungOneUI font for the English language works on TV devices but doesn't on watches. I'm still not sure why the behavior is different on watch devices. I could dig deeper but I think the current solution is enough to solve the issue.

Fixes https://github.com/flutter-tizen/flutter-tizen/issues/72.

### CODE REVIEW

File: third_party/txt/src/txt/platform_linux.cc [hunk 1]
```diff
@@ -13,14 +13,40 @@
 namespace txt {
 
 std::vector<std::string> GetDefaultFontFamilies() {
-  return {"SamsungOneUI", "BreezeSans", "Ubuntu", "Cantarell", "DejaVu Sans", "Liberation Sans", "Arial"};
+  return {
+      "SamsungOneUI",           "SamsungOneUIArabic",
```
Comment (bwikbs) [inline third_party/txt/src/txt/platform_linux.cc:17]: Where did you get these lists? wearable devices?

Comment (HakkyuKim) [reply third_party/txt/src/txt/platform_linux.cc:17]: Wearable and TV devices. 

Actually, @swift-kim mentioned that these fonts are not enough(need to add fonts in emulators), so this PR still needs update.

**Comment by HakkyuKim** [comment]
I couldn't solve the issue using fontconfig, I will leave it as future work.

**Review by swift-kim — APPROVED** [review]
No meaningful difference in performance.
```sh
Application                              STIME     PSS     RSS
com.example.font_original                  483   37887   53086
com.example.font_pr                        489   37888   53087
```
(measured on rpi4-arm64 with a counter sample)

**Comment by bbrto21** [comment]
@HakkyuKim  I am curious if we can use this plugin without FontConfig. Can you check it?
  https://pub.dev/packages/google_fonts

**Review by bbrto21 — APPROVED** [review]

**Comment by swift-kim** [comment]
It seems I did something wrong in the measurement above. I measured the app startup time again on my Galaxy Watch (R800) and the result is as follows:

```
Application                              STIME     PSS     RSS
com.example.font_original                  818   25965   53849
com.example.font_pr                       4643   25426   53408
```

Apparently there is a serious performance problem triggered by this change.

**Pull request merged**